src/dev/i18n/utils.test.js

```diff
@@ -45,17 +45,6 @@ describe('i18n utils', () => {
     expect(formatJSString('Test\\\n str\\\ning')).toEqual('Test string');
   });
 
-  test('should escape linebreaks', () => {
```

Comment:
__nit:__ better leave this test case but rename it to `should not escape linebreaks`